drivers/bluetooth/Kconfig

```diff
@@ -24,6 +24,27 @@ if BT_CUSTOM
 # Insert here any custom (non-HCI) offload drives
 endif
 
+config BT_NRF52840_PCA10090_GPIO_RESET
+	bool "Use nRF52840 PCA10090 GPIO reset pin"
+	depends on BT_H4
+	depends on (BOARD_NRF9160_PCA10090 || BOARD_NRF9160_PCA10090NS)
+	default y
+	help
+	  Use a GPIO pin to reset the nRF52840 controller and let it wait
+	  until all bytes traveling to the H4 device have been received
+ 	  and drained, thus ensuring communication can begin correctly.
+
+if BT_NRF52840_PCA10090_GPIO_RESET
+
+config BT_NRF52840_PCA10090_GPIO_RESET_PIN
```

Comment:
This belongs in the board Kconfig, and in fact you already have it there, so please remove this option.

Reply:
The host and the controller run on two different boards; the host does not have the macro selecting the pin defined (that's in `nrf52840_pca10090` board Kconfig), it needs its own.

Reply:
Right, but it still should be in the board Kconfig, not this one